backend/routers/slack.router.js

```diff
@@ -111,4 +114,63 @@ async function findProject(req, res) {
       });
     });
 }
-module.exports = router;
+
+//============= SLACKBOT FOR SENDING NEW USERS MESSAGES AND REMINDERS ==============
+
+//FETCH LIST OF NEW USERS (LESS THAN 30 DAYS) 
+async function getUsers() {
+  //querying to find users that were created in the last 30 days
+  const currentDate = new Date();
+  const thirtyDaysPrior = new Date().setDate(currentDate.getDate() - 30);
+  const thirtyDaysISO = new Date(thirtyDaysPrior).toISOString();
+  
+  const query = {createdDate: {$gte: thirtyDaysISO }}
+  
+  const users = await User.find(query);
+  return users;
+}
+
+console.log("DM Chat:", findAndDmChat());
+
+// MESSAGES LIST OF USERS WE GOT FROM getUsers() VIA SLACKBOT
+async function findAndDmChat() {
+  try {
+    const potentialSlackUsers = await getUsers();
+    const actualSlackUsers = [];
+
+    //find users who are in our slack
+    for (let user of potentialSlackUsers) {
+      console.log("Email:", user.email)
+      try {
+        let result = await app.client.users.lookupByEmail({
```

Comment:
Although I know that I was the direct cause of this implementation, please take a gander at this Google best practices on async/await for slightly refactoring this section for the better. https://developers.google.com/web/fundamentals/primers/async-functions#example_outputting_fetches_in_order

I do know that you are busy with things, so just let me know if you are too busy for that.

Reply:
Thanks for reviewing Nick! I will be a bit slammed in the coming weeks in terms of scheduling, maybe I can create an issue to improve on my PR?